### DISCUSSION TITLE
Why use NextJS if you can just use do it yourself?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#124219] [Other Feature Feedback, Questions, & Ideas]
I'm very new to all this programming stuff and don't really know much about it. The first language I learned was HTML and CSS. Now people tell me I should use a framework called NextJS. Why would I need a framework if I can just do it myself? Same with the Tailwind CSS Framework, I know people like to say its easier but is it really? Does it look better on the resume?

**Comment by BabaC0des** [comment]
Why Use Next.js?
Next.js provides server-side rendering and static site generation for better performance and SEO, automatic routing, easy API routes, built-in CSS and JS support, and enhances the overall developer experience

Why Use Tailwind CSS?
Tailwind CSS offers a utility-first approach for fast and flexible styling, ensures design consistency, supports responsive design, and is highly customizable.

I highly recommend you looking into it, it really is a great tool. It makes it easier and if you use it properly it'll make your life much easier.

**Reply by ghost** [reply]
Could you help me understand it better?

**Reply by BabaC0des** [reply]
Sure, contact me

**Comment by ebndev** [comment]
Hi @Napolisation & @milaasHD, thanks for being a part of the GitHub Community!

Please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
